Resources/doc/index.rst

```diff
@@ -125,6 +125,7 @@ The following annotations are defined by the bundle:
 
 This example shows all the available annotations in action::
 
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
```

Comment:
Actually, I would remove the `extends Controller` part instead as it's not relevant for this example.